### PR TITLE
CC-1181: Configure new parameters for pahmaUoCbyStudentVisits report

### PR DESCRIPTION
**Pull request by lkvoong**
New report parameters:
- StaffName
- VisitStartDate
- VisitEndDate

### CODE REVIEW

File: src/plugins/invocables/report/pahmaUoCbyStudentVisits/fields.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        view: {
+          type: CompoundInput,
+        },
+      },
+      StaffName: {
+        [config]: {
+          messages: defineMessages({
+            name: {
+              id: 'field.pahmaUoCbyStudentVisits.StaffName.name',
+              defaultMessage: 'Staff Name',
```
Comment (cesarvh) [inline src/plugins/invocables/report/pahmaUoCbyStudentVisits/fields.js]: Hey Lam! Forgot to mention this, but by convention we use sentence case for messages. Please change to "Staff name"

File: src/plugins/invocables/report/pahmaUoCbyStudentVisits/fields.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            },
+          },
+        },
+      },
+      VisitStartDate: {
+        [config]: {
+          messages: defineMessages({
+            name: {
+              id: 'field.pahmaUoCbyStudentVisits.VisitStartDate.name',
+              defaultMessage: 'Visit Start Date',
```
Comment (cesarvh) [inline src/plugins/invocables/report/pahmaUoCbyStudentVisits/fields.js]: Visit start date

File: src/plugins/invocables/report/pahmaUoCbyStudentVisits/fields.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+            type: DateInput,
+          },
+        },
+      },
+      VisitEndDate: {
+        [config]: {
+          messages: defineMessages({
+            name: {
+              id: 'field.pahmaUoCbyStudentVisits.VisitEndDate.name',
+              defaultMessage: 'Visit End Date',
```
Comment (cesarvh) [inline src/plugins/invocables/report/pahmaUoCbyStudentVisits/fields.js]: Visit end date

**Review by cesarvh — CHANGES_REQUESTED** [review]
Hey Lam! Just a note on the sentence casing for these

File: src/plugins/invocables/report/pahmaUoCbyStudentVisits/fields.js (hunk 1 above)
Comment (lkvoong) [reply src/plugins/invocables/report/pahmaUoCbyStudentVisits/fields.js]: Fixed. Thank you!

File: src/plugins/invocables/report/pahmaUoCbyStudentVisits/fields.js (hunk 2 above)
Comment (lkvoong) [reply src/plugins/invocables/report/pahmaUoCbyStudentVisits/fields.js]: Fixed. Thank you!

File: src/plugins/invocables/report/pahmaUoCbyStudentVisits/fields.js (hunk 3 above)
Comment (lkvoong) [reply src/plugins/invocables/report/pahmaUoCbyStudentVisits/fields.js]: Fixed. Thank you!

**Pull request merged**